drltrace/CMakeLists.txt

```diff
@@ -109,3 +109,12 @@ endif ()
 install(TARGETS drltrace DESTINATION "${INSTALL_BIN}"
   PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
   WORLD_READ WORLD_EXECUTE)
+
+##################################################
+# drltrace test
+
+if (BUILD_TOOL_TESTS)
+  get_target_property(app_path drsyscall_app LOCATION${location_suffix})
+  get_target_property(drltrace_path drltrace LOCATION${location_suffix})
+  add_test(drltrace ${drltrace_path} -- ${app_path})
```

Comment:
There is a strange error at AppVeyor
Build started
git clone -q --depth=10 https://github.com/DynamoRIO/drmemory.git C:\projects\drmemory
fatal: early EOF
fatal: The remote end hung up unexpectedly
fatal: index-pack failed
error: RPC failed; curl 56 SSL read: error:00000000:lib(0):func(0):reason(0), errno 10054
Command exited with code 128